apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -54,7 +62,10 @@ export function InstructorAssessmentSettings({
                     id="title"
                     name="title"
                     value="${resLocals.assessment.title}"
-                    disabled
+                    ${resLocals.authz_data.has_course_permission_edit &&
+                    !resLocals.course.example_course
```

Comment:
Nit: you might consider lifting this conditional to a variable like `canEdit` that you can then use in all these places.